src/app/modules/color-scheme/color-scheme-popup/color-scheme-popup.component.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { Shallow } from 'shallow-render';
+
+import { ColorScheme, ColorSchemePopupComponent } from './color-scheme-popup.component';
+import { ColorSchemePopupModule } from './color-scheme-popup.module';
+
+describe('FiltersContentComponent', () => {
+  let shallow: Shallow<ColorSchemePopupComponent>;
+  const testScheme: ColorScheme = {
+    type: 'discrete',
+    name: 'test',
+    colors: ['color1', 'color2'],
+    positions: [0, 1]
+  };
+
+  let mockHTMLElement: jasmine.SpyObj<HTMLElement>; // tslint:disable-line:prefer-const
```

Comment:
This is never set! All tests using this are incorrect (even if they pass)!
Also never do `// tslint:disable-line:prefer-const`
It is often an indicator you have missed setting the value before using it!